app/models/status.rb

```diff
@@ -273,7 +273,7 @@ def non_sensitive_with_media?
   end
 
   def reported?
-    @reported ||= Report.where(target_account: account).unresolved.where('? = ANY(status_ids)', id).exists?
+    @reported ||= Report.where(target_account: account).where('? = ANY(status_ids)', id).exists?
```

Comment:
I wanted the definition of `status.reported?` to be `true` even if the Report against the Status has already been resolved.

The [Admin Status Policy](https://github.com/MozillaSocial/mastodon/blob/20952f9a0d46603c206cafb458ed62adb98fe725/app/policies/admin/status_policy.rb#L15) lets the Admin API "show (GET)" Statuses that would otherwise not be visible to them, if the status is reported.

Since we auto-resolve reports in the bridge, we need `reported?` to persist as `true` even after the report is resolved.

I don't think this changes the intention too much, so I think upstream will be ok with this too. If not, it's easy to remove for the PR we do there.